Data/DataModel/Submission.cs

```diff
@@ -73,6 +73,21 @@ public string SubmissionText
         /// </summary>
         public bool? FreeformAccepted { get; set; }
 
+        /// <summary>
```

Comment:
I will admit to some anxiety about changing the DB this close to the event with no betas in between. How confident are you?

Reply:
Discussed offline and we're taking precautions including DB backup, extra code review and extra testing